packages/core/core-flows/src/cart/workflows/refresh-cart-items.ts

```diff
@@ -101,62 +101,59 @@ export const refreshCartItemsWorkflow = createWorkflow(
           },
         },
       }).config({ name: "fetch-variants" })
-    })
 
-    validateVariantPricesStep({ variants })
-
-    const validate = createHook("validate", {
-      input,
-      cart,
-    })
```

Comment:
thought: Was this intentionally removed?

Reply:
Yeah, this was intentional. That hook is present in all workflows where this one is called as a step.
Here we are not mutating the cart per se, I believe the hook here is not valid.